earth2studio/data/arco.py

```diff
@@ -258,6 +259,22 @@ async def fetch_array(self, time: datetime, variable: str) -> np.ndarray:
             logger.error(f"variable id {variable} not found in ARCO lexicon")
             raise e
 
+        # Handle 6-hour precipitation accumulation
```

Comment:
Hey @NickGeneva , Is there a better place I should put this? Its tough because it changes what the API call will be.

Reply:
tp06 is in wb2,
We shouldnt add this to arco directly. We should only handle unit conversions, nothing more because we don't make those judgment calls. Can uplift this to an example if needed.

Maybe we need a e2studio mega data source that takes in several sources and does this type of stuff for people...